builtin/providers/aws/provider.go

```diff
@@ -215,6 +215,9 @@ func Provider() terraform.ResourceProvider {
 			"aws_lambda_permission":                        resourceAwsLambdaPermission(),
 			"aws_launch_configuration":                     resourceAwsLaunchConfiguration(),
 			"aws_lb_cookie_stickiness_policy":              resourceAwsLBCookieStickinessPolicy(),
+			"aws_elb_load_balancer_policy":                 resourceAwsLoadBalancerPolicy(),
```

Comment:
i'm happy with the names as they stand, would be fine with renaming them to 

```
aws_load_balancer_policy
aws_load_balancer_backend_server_policy
aws_load_balancer_listener_policy
```

if deemed appropriate.